tests/TestCase.php

```diff
@@ -44,4 +46,22 @@ protected static function getHttpBinUri(): string
 
 		return self::$httpBinUri;
 	}
+
+	/**
+	 * Returns example message values loaded from message.json
+	 */
+	protected static function getMessageExample(): stdClass
+	{
+		$data = (string) file_get_contents(__DIR__ . '/TestAssets/message.json');
```

Comment:
Could include new helper function to ensure errors are raised to make it easier to debug for missing fixtures.

```
    protected static function loadFixture(string $name): string
    {
        $fixturePath = __DIR__.'/Fixtures/'.$name;
        $contents = file_get_contents($fixturePath);

        if (false === $contents) {
            throw new RuntimeException(sprintf('Unable to load fixture: %s', $fixturePath));
        }

        return $contents;
    }
```

Reply:
done. much better than having two methods.